R/modelling.R

```diff
@@ -306,7 +307,7 @@ fit_seromodel <- function(
 }
 
 
-#' Function that generates a data.frame containing the age of each cohort
+#' Generate data.frame containing the age of each cohort
```

Comment:
data frame (I think generally we should stick to proper language where possible)

Reply:
Thanks, I just corrected this across the package documentation.

Reply:
Done